dot/utils.go

```diff
@@ -100,7 +100,7 @@ func NewTestGenesisAndRuntime(t *testing.T) (filename string) {
 	runtimeData, err := os.ReadFile(filepath.Clean(runtimeFilePath))
 	require.Nil(t, err)
 
-	gen := newTestGenesis(t)
+	gen := NewTestGenesis(t)
```

Comment:
can we change this back to being unexported? 🤔

Reply:
This is exported, it's being used in cmd/config_test.go.

Reply:
How was it unexported before then? 🤔 
Otherwise would there be a way to copy pasta it in `cmd` so the cmd unit tests don't depend on `dot/utils.go`?